python/ucxx/_lib_async/endpoint.py

```diff
@@ -214,6 +249,44 @@ async def send_obj(self, obj, tag=None):
         await self.send(nbytes, tag=tag)
         await self.send(obj, tag=tag)
 
+    async def am_recv(self):
+        """Receive from connected peer via active messages."""
+        if not self._ep.am_probe():
+            self._ep.raise_on_error()
+            if self.closed():
+                raise UCXCloseError("Endpoint closed")
+
+        # Optimization to eliminate producing logger string overhead
+        if logger.isEnabledFor(logging.DEBUG):
+            log = "[AM Recv #%03d] ep: %s" % (
+                self._recv_count,
+                hex(self.uid),
+            )
+            logger.debug(log)
+
+        self._recv_count += 1
+
+        req = self._ep.am_recv()
+        await req.wait()
+        buffer = req.get_recv_buffer()
+
+        if logger.isEnabledFor(logging.DEBUG):
+            log = "[AM Recv Completed #%03d] ep: %s, nbytes: %d, type: %s" % (
+                self._recv_count,
+                hex(self.uid),
+                buffer.nbytes,
+                type(buffer),
+            )
+            logger.debug(log)
```

Comment:
And here... (Note also using `"0x%x"` to format the uid as hex, rather than calling `hex(self.uid)`).

Reply:
Done in https://github.com/rapidsai/ucxx/pull/48/commits/bd1787f0b17d505f6e4ec60037fc605d057bcd01